test/Common.cpp

```diff
@@ -103,6 +103,8 @@ CommonOptions::CommonOptions(std::string _caption):
 		("no-smt", po::bool_switch(&disableSMT), "disable SMT checker")
 		("optimize", po::bool_switch(&optimize), "enables optimization")
 		("enforce-via-yul", po::bool_switch(&enforceViaYul), "Enforce compiling all tests via yul to see if additional tests can be activated.")
+		("enforce-gas-cost", po::bool_switch(&enforceGasTest), "Enforce gas cost expectations in semantic tests.")
```

Comment:
Shouldn't this rather be named something like `fail-on-gas-cost-mismatch`? What is this enforcing?

Reply:
Or `check-gas-cosst`?

Reply:
I thought we said to have something like `--gas-tests-threshold <n>` where n specifies the minimum required gas above which it is included in the test.

Reply:
with enforcing it is going to fail the tests that use more than `X` gas and force us to add this costs to the tests... `X` can be set with the option below.
If you think it is not useful, I will remove it completely.